sda/cmd/api/swagger_v1.yml

```diff
@@ -0,0 +1,410 @@
+openapi: 3.0.3
+info:
+  title: SDA admin API
+  version: "1.0"
+  description: This is the admin API for the sensitive data archive.
+paths:
+  /c4gh-keys/add:
+    post:
+      description: Registeres an crypt4gh public key hash in the database
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/C4ghKeyAdd"
+      responses:
+        "200":
+          description: Successful operation.
+        "400":
+          description: Bad payload
+        "401":
+          description: Authentication failure.
+        "409":
+          description: Key hash already exists in the database.
+        "500":
+          description: Internal application error.
+  /c4gh-keys/deprecate/{keyHash}:
+    post:
+      description: Deprecate a given key hash
+      parameters:
+        - in: path
+          name: keyHash
+          schema:
+            type: string
+          required: true
+      responses:
+        "200":
+          description: Successful operation
+        "400":
+          description: Bad key hash
+        "401":
+          description: Authentication failure
+        "500":
+          description: Internal application error
+  /c4gh-keys/list:
+    get:
+      description: Returns a list of all registered encryptio ns keys and thei statuses.
```

Comment:
```suggestion
      description: Returns a list of all registered encryptions keys and their statuses.
```

Reply:
encryption keys without the s in encryptions?